code/game/objects/items/devices/radio/headset.dm

```diff
@@ -128,6 +128,14 @@
 	icon_state = "sec_headset_alt"
 	item_state = "sec_headset_alt"
 
+/obj/item/radio/headset/headset_iaa
+	name = "internal affairs bowman headset"
+	desc = "This is used by your elite legal team. Protects ears from flashbangs."
+	flags = EARBANGPROTECT
```

Comment:
Not sure if IAA should be flashbang proof. The NT rep ain't.

Reply:
They already were. Previously they got a sec headset and I didn’t want to balance change. When I brought this up the rather valid point I heard was that flash bangs get used in sec and IAA is in sec a lot.

I’m not against changing it though.

Reply:
To that point and bringing in something Kyet said last which I happen to agree with - the IAA _shouldn't_ be just hanging in their office which just happens to be located at the Brig. They should be moving around the station, checking in on departments and other things. Flashbangs suck, sure, but making their headsets immune just because of where they are, I don't know. I support removing the flashbang proofing, especially since this feels like a step in a bigger picture of remodeling the IAA role. I wouldn't be surprised if the future has their office being moved closer to the NT Rep once the map freeze is lifted.